### PR TITLE
chore(ci): CI with Github Actions

### PR DESCRIPTION
**Pull request by strophy**
## Issue being fixed or feature implemented

We experiencing several problems with Travis CI. It is slow and does not allow to restart builds.

## What was done?

- Introduce Github Action workflows for running tests and releasing.
- Delete Travis CI script
- Update badges
- Fix invalid readme syntax

## How Has This Been Tested?

With Github Actions

## Breaking Changes

None

## Checklist:

- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have added or updated relevant unit/integration/functional/e2e tests
- [x] I have made corresponding changes to the documentation

**For repository code-owners and collaborators only**
- [x] I have assigned this pull request to a milestone

### CODE REVIEW

File: .github/workflows/test_and_release.yml [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+      - name: Publish NPM package
+        uses: JS-DevTools/npm-publish@v1
+        with:
+          token: ${{ secrets.NPM_TOKEN }}
+          tag: ${{ steps.tag.outputs.result }}
+
+  release-docker:
+    name: Release Docker image
+    runs-on: ubuntu-20.04
+    needs: test
```
Comment (shumkov) [inline .github/workflows/test_and_release.yml]: release npm

**Review by shumkov — CHANGES_REQUESTED** [review]

File: .github/workflows/test_and_release.yml (hunk 1 above)
Comment (strophy) [reply .github/workflows/test_and_release.yml]: Why? This should run in parallel to the `release-npm` job. The `build-args` needed in the Dockerfile are [passed here already](https://github.com/dashevo/insight-api/blob/github-actions/.github/workflows/test_and_release.yml#L143).

Comment (shumkov) [reply .github/workflows/test_and_release.yml]: It installs npm package which should be published before. Please take a look at Docketfile.

Comment (strophy) [reply .github/workflows/test_and_release.yml]: I think I see what you mean now, it is [these lines](https://github.com/dashevo/insight-api/blob/master/docker/Dockerfile#L25) that require the NPM release to be pushed first right? What does `dashcore-node` actually do, it looks like JS implementation of dashd? How is this different to the `dash` package?

**Review by shumkov — APPROVED** [review]
👍

File: .github/workflows/test_and_release.yml (hunk 1 above)
Comment (shumkov) [reply .github/workflows/test_and_release.yml]: `dash` package is a light client that uses DAPI to fetch, verify, and broadcast data to Dash Network. `dashcore-node` represents a full node.

**Pull request merged**